app/views/orders/show.html.erb

```diff
@@ -30,14 +31,23 @@
     </tr>
   <% @order.item_orders.each do |item_order|%>
     <tr>
+      <% if item_order.item.merchant_id == @merchant.id %>
     <section id = "item-<%=item_order.item_id%>">
-        <td><p><%=link_to item_order.item.name, "/items/#{item_order.item_id}"%></p></td>
-        <td><p><%= link_to item_order.item.merchant.name, "/merchants/#{item_order.item.merchant.id}"%></p></td>
-        <td><p><%= number_to_currency(item_order.price)%></p></td>
-        <td><p><%= item_order.quantity%></p></td>
-        <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
-      </section>
+        <% if item_order.status == 0 && item_order.quantity <= item_order.item.inventory %>
+          <td><p><%= link_to "Fulfill Order", "/item_orders/#{item_order.id}/fulfill"%></p></td>
+        <% elsif item_order.quantity >= item_order.item.inventory %>
+          <td><p>This order cannot be fulfilled due to lack of inventory.</p></td>
+        <% else %>
+          <td><p>Order has been fulfilled.</p></td>
+        <% end %>
```

Comment:
These look like flash messages that belong in the controller. We might be breaking some MVC rules here.

Reply:
That's something that should display for many different item_orders on the same page all at once, but each item_order may have a different status so I added a column to the css table to display whether that item is fulfilled, can be fulfilled or cannot be fulfilled